x/epochs/keeper/abci.go

```diff
@@ -24,24 +24,29 @@ func (k Keeper) BeginBlocker(ctx sdk.Context) {
 		shouldEpochEnd := ctx.BlockTime().After(epochEndTime) && !shouldInitialEpochStart && !epochInfo.StartTime.After(ctx.BlockTime())
 
 		epochInfo.CurrentEpochStartHeight = ctx.BlockHeight()
+		identifier, exists := types.DurationToIdentifier[epochInfo.Duration]
+		if !exists {
+			// continue
+			return false
+		}
 
 		switch {
 		case shouldInitialEpochStart:
 			epochInfo.StartInitialEpoch()
```

Comment:
This issue remains: https://github.com/tharsis/evmos/pull/553#discussion_r874737391
> can this panic? wonder why is the code scanning tool giving a warning now

I don't know yet why. I replaced
```
identifier, exists := types.DurationToIdentifier[epochInfo.Duration]
		if !exists {
			// continue
			return false
		}
```
with `identifier := ""`, to see if there is any change: there was not (same code analysis panics). So it is not the map or identifier. Seems related to `epochInfo`.